sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,23 +1,36 @@
 # Release History
 
-## 1.14.0-beta.3 (Unreleased)
+## 1.14.0 (2024-10-03)
 
 ### Features Added
 
+- Added new constructor for `Azure::Core::Context` that takes a `std::chrono::system_clock::time_point` deadline. This enables creating a new context directly with a deadline.
```

Comment:
This isn't accurate, the new ctor accepts an Azure::DateTime.

https://github.com/Azure/azure-sdk-for-cpp/blob/171386533aba7cee966a50daff9735fe2431a1db/sdk/core/azure-core/inc/azure/core/context.hpp#L209

Also, nit:
Added **a** new constructor...
Instead of
Added new constructor...

Reply:
@LarryOsterman, you've written that entry, if you'd rephrase it, could you please post a suggestion comment with the edits? (Or, if you'd like to keep it as-is, could you please also reply?)

Reply:
Whatever you feel comfortable with. I think that the comment should be accurate, and I have no issues with the grammar suggestion.

Reply:
I probably should have, in this case.